project/src/pages/welcome-page/welcome-page.tsx

```diff
@@ -14,10 +16,21 @@ type WelcomePageProps = {
 
 function WelcomePage({offerCount, offers}: WelcomePageProps): JSX.Element {
 
+  const dispatch = useAppDispatch();
+  const currentCity = useAppSelector((state) => state.city);
+  const currentOffers = useAppSelector((state) => state.offers);
   const [selectedOffer, setSelectedOffer] = useState({});
 
+  useEffect(() => {
+    dispatch(addOffers(
+      offers.filter((offer) => offer.city.name === currentCity.title)
+    ));
+  }, [currentCity, dispatch, offers]);
+
+  const onClickCity = (city: City) => {dispatch(clickCity(city));};
```

Comment:
название обработчика поменяй на 
```suggestion
  const onCityClick = (city: City) => {dispatch(clickCity(city));};
```